pandas/plotting/_core.py

```diff
@@ -2629,110 +594,159 @@ def boxplot_frame_groupby(grouped, subplots=True, column=None, fontsize=None,
     >>> grouped = df.unstack(level='lvl1').groupby(level=0, axis=1)
     >>> boxplot_frame_groupby(grouped, subplots=False)
     """
-    _raise_if_no_mpl()
-    _converter._WARN = False
-    if subplots is True:
-        naxes = len(grouped)
-        fig, axes = _subplots(naxes=naxes, squeeze=False,
-                              ax=ax, sharex=sharex, sharey=sharey,
-                              figsize=figsize, layout=layout)
-        axes = _flatten(axes)
-
-        from pandas.core.series import Series
-        ret = Series()
-        for (key, group), ax in zip(grouped, axes):
-            d = group.boxplot(ax=ax, column=column, fontsize=fontsize,
-                              rot=rot, grid=grid, **kwds)
-            ax.set_title(pprint_thing(key))
-            ret.loc[key] = d
-        fig.subplots_adjust(bottom=0.15, top=0.9, left=0.1,
-                            right=0.9, wspace=0.2)
+    plot_backend = _get_plot_backend()
+    return plot_backend.boxplot_frame_groupby(
+        grouped, subplots=subplots, column=column, fontsize=fontsize, rot=rot,
+        grid=grid, ax=ax, figsize=figsize, layout=layout, sharex=sharex,
+        sharey=sharey, **kwds)
+
+
```

Comment:
_get_plot_backend could be here (ignore my previous comment), to organize things. can you add an explanation block on what is the purpose of all this (also blank line separate before comments)